### ISSUE TITLE
Traces GUI more user friendly

### ISSUE THREAD

**Issue by janmaslik** [SigNoz/signoz#2172]
Hi guys, nice work.
Im coming with few hints.
I would like reach potential of traces. 

- [x] - Possibility to save/load filters in Traces and Logs . This view has a lot of features. Just i would like to have possibility to save it . And reuse again later with actual timeframe (or use it as base for next search )
- [ ] #8284
![image](https://user-images.githubusercontent.com/30992729/216367349-78c89973-e1b4-4eb7-8d9b-3085b3667f38.png)
- [x] - Navigate from **home / trace / (spanid)**  (previous print screen) to related logs which belongs to the span

Basically use informations on the page also as navigation links

**Comment by makeavish** [comment]
Thanks for opening the issue @janmaslik 
These suggestions looks very useful, will integrate these features in future.

**Comment by makeavish** [comment]
Created new issue here: https://github.com/SigNoz/signoz/issues/8284
